[2.8.6] added doc for Rancher CLI login with AzureAD (#1311)
rancher/rancher-docs: docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/authentication-config/configure-azure-ad.md

:::

#### 4. Allow Public Client Flows

> To login from Rancher CLI you will need to allow Public Client Flows.

Reviewer: Per the check, the word "will" should be removed.

Author: Thanks @crobby, before fixing it, is the rest ok? :D

> To login from Rancher CLI you will need to allow Public Client Flows.

Reviewer 2: ```suggestion
To login from Rancher CLI you must allow Public Client Flows.
```

> 1. From the navigation pane on, select **Authentication**.

Reviewer 2: ```suggestion
1. From the left navigation menu, select **Authentication**.
```

Author: Thanks @martyav. I just want to highlight that this was copy-pasted from line 104. Should I update also that?

> 1. In the **Advanced Settings** enable the **public client flows**.

Reviewer 2: ```suggestion
1. Under **Advanced Settings**, enable **public client flows**.
```
